diff --git a/target/linux/qualcommax/files/ipq8072-xiaomi-ax3600.dts b/target/linux/qualcommax/files/ipq8072-xiaomi-ax3600.dts
index xxxxxxx..yyyyyyy 100644
--- a/target/linux/qualcommax/files/ipq8072-xiaomi-ax3600.dts
+++ b/target/linux/qualcommax/files/ipq8072-xiaomi-ax3600.dts
@@ -124,14 +124,8 @@
 				partition@0400000 {
 					label = "rootfs";
 					reg = <0x0400000 0x7000000>;
-				};
-
-				partition@7400000 {
-					label = "rootfs_1";
-					reg = <0x7400000 0x7000000>;
 				};
-
-				partition@e400000 {
+				partition@7400000 {
 					label = "ubi";
 					reg = <0x7400000 0x8c00000>;
 				};
